specification/library-guidelines.md

```diff
@@ -34,35 +34,69 @@ Applications that use third-party libraries that are instrumented with OpenTelem
+The API dependency contains a minimal implementation of the API. When no other implementation is explicitly included in the application no telemetry data will be collected. Here is what active components look like in this case:
 
 ![Minimal Operation Diagram](language-library-minimal.png)
 
 It is important that values returned from this minimal implementation of API are valid and do not require the caller to perform extra checks (e.g. createSpan() method should not fail and should return a valid non-null Span object). The caller should not need to know and worry about the fact that minimal implementation is in effect. This minimizes the boilerplate and error handling in the instrumented code.
 
 It is also important that minimal implementation incurs as little performance penalty as possible, so that third-party frameworks and libraries that are instrumented with OpenTelemetry impose negligible overheads to users of such libraries that do not want to use OpenTelemetry too.
 
-### SDK Implementation
+## SDK Implementation
 
-The SDK implementation is a separate (optional) dependency. When it is plugged in it substitutes the minimal implementation that is included in the API package (exact substitution mechanism is language dependent). Here is what active components will look like in this case:
+The SDK implementation is a separate (optional) dependency. When it is plugged in it substitutes the minimal implementation that is included in the API package (exact substitution mechanism is language dependent).
+
+SDK implements core functionality that is required for translating API calls into telemetry data that is ready for exporting. Here is the non-comprehensive list of core functionality:
+
+- Scoped span operations.
+- Context propagation.
+- Span and trace id generation.
+- Span start/end time tagging.
+
+Here is how OpenTelemetry components look like when the SDK is enabled:
 
 ![Full Operation Diagram](language-library-full.png)
 
-It is recommended to decouple common parts of the implementation from protocol-specific Telemetry Exporters. Library designers are encouraged to specify an internal Telemetry Exporter API that defines how protocol-independent and protocol-dependant parts interact. The boundary and design goals for such API will be:
+SDK defines an [Exporter interface](sdk-exporter.md). Protocol-specific exporters that are responsible for sending telemetry data to backends must implement this interface.
+
+SDK also includes optional helper exporters that can be composed for additional functionality if needed. It is recommended to implement the following helper exporters: queue, multi-exporter, retrier, etc. See [example usage below](#example-usage-of-helper-exporters).
+
+Library designers need to define the language-specific Exporter interface based on [this generic specification](sdk-exporter.md). 
+
+### Composable Helper Exporters
```

Comment:
I would move this section in the exporter file, I think we should not enter into too many details in this guideline file.

Reply:
I agree. I also had a feeling that this is not the right place. Removing.